src/routes/safe/components/Apps/index.tsx

```diff
@@ -409,15 +413,15 @@ function Apps({ closeModal, closeSnackbar, enqueueSnackbar, openModal }) {
           </Centered>
         </Card>
       )}
-      <Centered>
+      <CenteredMT className="mt">
```

Comment:
Please remove the classname and good to go :)

Reply:
done!